src/dbos-runtime/cli.ts

```diff
@@ -26,6 +26,7 @@ export interface DBOSCLIStartOptions {
   configfile?: string;
   appDir?: string;
   appVersion?: string | boolean;
+  logWhileParsing?: boolean;
```

Comment:
How is it specified? I don't see a new CLI option. If we want to keep this, maybe change this to `verbose`?

Reply:
Added a comment